dist/newrelic/newrelic.yml

```diff
@@ -0,0 +1,302 @@
+# This file configures the New Relic Agent.  New Relic monitors
```

Comment:
Replies..
1. We are not throwing any configs away.  The existing configs are in this change file.  New configs in this file are in addition to the existing configs.   I do see that enable_auto_app_naming is different so I will update to 'true' which matches existing config.

2. The setting in the file are just defaults settings which can be overriden in the new relic UI.  Keeping them in the file allows you to more easily change defaults and not have to reference the docs to determine what the defaults are for each config.  It continue to work the same as the existing config file.  You don't need to redeploy bridge if you set these configs in the new relic UI.  Anyways if you are still opposed to this then I can remove the configs and make this match the existing heroku config file.